WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -368,6 +373,19 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         listPopup.show();
     }
 
+    private void updatePostUploadProgressBar(ProgressBar view, PostModel post) {
+        if (UploadService.isPostUploadingOrQueued(post)) {
```

Comment:
That'd be nice! Do you know how to set that attribute programmatically? Can't find that in the API docs